lib/dialects/mysql/connection-manager.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
```

Comment:
Only case for `SequelizeErrors` is changed, remove changes from this file

Reply:
It was for consistancy, it's the only file where sequelizeErrors is used with an upper first letter. I'm reverting.

Reply:
Leave it